README.rst

```diff
@@ -193,7 +193,7 @@ PyQT embedding
     win.show()
     sys.exit(app.exec_())
 
-PyGtk embedding
+PyGObject embedding
```

Comment:
I believe the next line should be edited to match the length of this header.

Reply:
Thanks for pointing that out :smiley: See 69337f8